lib/config/dry_validation_requirements.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
```

Comment:
Why not keeping that?

Reply:
It's not in the rest of the repo, no strong opinion.